packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -54,15 +54,9 @@ export const Alert = React.forwardRef<HTMLDivElement, Props>(
     const ariaLabel = restProps['aria-label'] || title;
 
     return (
-      <div
-        ref={ref}
-        className={cx(styles.wrapper, className)}
-        data-testid={selectors.components.Alert.alertV2(severity)}
-        role={role}
-        aria-label={ariaLabel}
-        {...restProps}
-      >
+      <div ref={ref} className={cx(styles.wrapper, className)} role={role} aria-label={ariaLabel} {...restProps}>
```

Comment:
I'm moving the data-testid from this outer div to the inner box so the `...restProps` passed to this component can reflect here. 

This allow us to use `data-testid` instead of `aria-label` in the `<Alert` component